### PR TITLE
Layout stars solution

### PR DESCRIPTION
**Pull request by WaveOfDandelions**
- [DEMO LINK](https://WaveOfDandelions.github.io/layout_stars/)
- [TEST REPORT LINK](https://WaveOfDandelions.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,58 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Stars title</title>
   </head>
   <body>
-    <h1>Stars</h1>
+    <main class="page">
+      <section class="stars page__stars">
+        <ul class="list stars__list stars--0">
```
Comment (Oskinavara) [inline src/index.html:13]: `<ul>` elements could be separated by an empty line.

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+.list__item {
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  list-style: none;
+  margin-right: 4px;
```
Comment (Oskinavara) [inline src/style.css:19]: Margin-right should not be added for the last element.

**Review by Oskinavara — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 2 above)
Comment (WaveOfDandelions) [reply src/style.css:19]: > Margin-right should not be added for the last element.

Dobrze, poprawie to

File: src/index.html (hunk 1 above)
Comment (WaveOfDandelions) [reply src/index.html:14]: > `<ul>` elements could be separated by an empty line.

Fixed

File: src/style.css (hunk 2 above)
Comment (WaveOfDandelions) [reply src/style.css:22]: > Margin-right should not be added for the last element.

Fixed

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+.stars--1 :nth-child(1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(1n) {
+  background-image: url(images/star-active.svg);
+}
```
Comment (choeqq) [inline src/style.css:31]: nicely done 👍

**Review by choeqq — APPROVED** [review]
All test are passing - good job!

**Pull request closed without merging**